ReText/window.py

```diff
@@ -95,6 +95,7 @@ def __init__(self, parent=None):
 		self.addToolBar(Qt.ToolBarArea.BottomToolBarArea, self.searchBar)
 		self.toolBar.setVisible(not globalSettings.hideToolBar)
 		self.editBar.setVisible(not globalSettings.hideToolBar)
+		self.urlPopup = urlPopup(self)
```

Comment:
Please remove it from here and make it an attribute of ReTextWebEnginePage.

Reply:
Good idea, nicely keeping it modular. Before, I thought it would be sufficient to have one instance of UrlPopup for one Retext window.

Latest implementation explicitly lets it stick to 'window': `def __init__(self, window):` (…and still has to (?))

Reply:
Off-topic / note by the way:

Meanwhile I had:
``` python
class UrlPopup(QLabel):
    def __init__(self, parent):
        super().__init__(parent.tab.p)
        self.page = parent
    def pop(…):
        windowBottom = self.page.tab.rect().bottom()
```
(only relevant lines, tab also has .rect())
--> Doing so I realized that _tab's bottom is well above the window's_ bottom! (But may be associated with various coordinate-systems window/tab)

Reply:
Tab's bottom is calculated from tab's top, and window's bottom from window's top. And there is menu bar, toolbar and tabs panel between window's top and tab's top. So they make the difference that you see.